test/test-execute/exec-emptydirectories.service

```diff
@@ -0,0 +1,22 @@
+[Unit]
+Description=Test for BindPaths= and BindReadOnlyPaths=
```

Comment:
Unimportant nit, but I think this is stale from whatever was used as a template in creating this...

Reply:
Oh, thanks. I will update.

Reply:
Updated.